apps/meteor/client/views/room/Header/Omnichannel/VoipRoomHeader.tsx

```diff
@@ -41,7 +41,7 @@ const VoipRoomHeader: FC<VoipRoomHeaderProps> = ({ slots: parentSlot, room }) =>
 				[context],
 			)}
 		>
-			<RoomHeader slots={slots} room={{ ...room, name: parseOutboundPhoneNumber(room.name) }} />
+			<RoomHeader slots={slots} room={{ ...room, name: parseOutboundPhoneNumber(room.fname as string) }} />
```

Comment:
I believe it makes more sense to allow the fn `parseOutboundPhoneNumber` to accept `string | undefined` instead of force casting a possible `undefined` to `string`.